examples/igrantio/Makefile

```diff
@@ -0,0 +1,28 @@
+.DEFAULT_GOAL := help
```

Comment:
Great that you are using Make! I really appreciate this as developer documentation and for having fast workflows :heart: